README.md

```diff
@@ -46,6 +46,21 @@ cd csharp\examples\simple
 dotnet run
 ```
 
+## Connecting to GCP GKE from local machine
+
+Start proxy to access kubernetes cluster:
+
+```bash
+$ kubectl proxy
+Starting to serve on 127.0.0.1:8001
+```
+
+In code use following config:
+
+```csharp
+var config = new KubernetesClientConfiguration {  Host = "http://127.0.0.1:8001" };
```

Comment:
The preferred way to connect is to use the config file on disk as follows:
```cs
            var k8SClientConfig = KubernetesClientConfiguration.BuildConfigFromConfigFile();
            IKubernetes client = new Kubernetes(k8SClientConfig);
```

https://github.com/kubernetes-client/csharp/blob/master/examples/namespace/Namespace.cs#L52